packages/main/src/themes/Button.css

```diff
@@ -65,8 +65,7 @@
 
 .ui5-button-icon {
 	font-size: var(--_ui5_button_icon_font_size);
-	height: 0;
-	top: -.5rem;
+	height: 100%;
```

Comment:
If we do this, what would happen to all of the IE11 fixes (for button/togglebutton)? Seems to work from what I can see (icon is still centered on IE)